packages/contracts-bedrock/src/governance/ProposalValidator.sol

```diff
@@ -225,7 +233,7 @@ contract ProposalValidator is OwnableUpgradeable, ReinitializableBase, ISemver {
         _setDistributionThreshold(_distributionThreshold);
 
         for (uint256 i = 0; i < _proposalTypes.length; i++) {
-            _setProposalTypeApprovalThreshold(_proposalTypes[i], _requiredApprovals[i]);
+            _setProposalTypeData(_proposalTypes[i], _proposalTypesData[i]);
```

Comment:
i am thinking, maybe worth having a check that `proposalTypes.length == _proposalTypesData.length` wdyt?